### PR TITLE
fix: Another try at fixing the notification channel crash from Google Play Console

### PR DESCRIPTION
**Pull request by makingthematrix**
I looked through the code of the commit which most probably causes the crashes and through Stack Overflow entries on similar issues. I found this: https://stackoverflow.com/a/51378281/2975925 which says that we shouldn't use localized strings as the notification channel ids, and I followed this advise in WebSocketService, and also I decided to bring back a line that ensures the notification channel is created when the app goes to foreground (just as it was before).

We are still not able to reproduce the crash 100% but I think this PR should help.

#### APK
[Download build #3284](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3284/artifact/build/artifact/wire-dev-PR3235-3284.apk)
[Download build #3286](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3286/artifact/build/artifact/wire-dev-PR3235-3286.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/services/websocket/WebSocketService.scala [hunk 1]
```diff
@@ -195,20 +196,28 @@ class WebSocketService extends ServiceHelper with DerivedLogTag {
...
+    notificationChannel
   }
 
-  private lazy val createNotificationChannel: Unit =
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-      val notificationManager = inject[NotificationManager]
-      notificationManager.createNotificationChannel(
-        returning(new NotificationChannel(ForegroundNotificationChannelId, getString(R.string.foreground_service_notification_name), NotificationManager.IMPORTANCE_LOW)) { ch =>
+  // this is the same code as in WireApplication.notificationChannel but I don't want to access it
+  // from here and the other way around as well (Maciek)
```
Comment (mejdoo) [inline app/src/main/scala/com/waz/services/websocket/WebSocketService.scala:203]: (Maciek)?

Comment (makingthematrix) [reply app/src/main/scala/com/waz/services/websocket/WebSocketService.scala:203]: yes, I wrote a comment there and left my name so if anyone wanted to do something about it, they can contact me 🙂

**Comment by makingthematrix** [comment]
build again

File: app/src/main/scala/com/waz/services/websocket/WebSocketService.scala (hunk 1 above)
Comment (mejdoo) [reply app/src/main/scala/com/waz/services/websocket/WebSocketService.scala:203]: we usually don't leave our names in the comments :)

**Review by mejdoo — APPROVED** [review]

**Pull request merged**